src/components/vr_cooperation/include/vr_cooperation/commands/json_notification.h

```diff
@@ -51,23 +51,22 @@ namespace commands {
 /**
  * @brief Base command class for notifications
  */
-class BaseCommandNotification : public Command {
+class JSONNotification : public Command {
```

Comment:
Where is VRModule* parent_; in this class? -> at line 100
Where is function SendNotification? -> at line 110